pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -349,10 +349,10 @@ private TableAuthorizationResult hasTableAccess(RequesterIdentity requesterIdent
   }
 
   /**
-   * Returns true if the QPS quota of the tables has exceeded.
+   * Returns true if the QPS quota of query tables, database or application has been exceeded.
    */
   private boolean hasExceededQPSQuota(@Nullable String database, Set<String> tableNames,
-      RequestContext requestContext) {
+                                      RequestContext requestContext) {
```

Comment:
nit: code style

Reply:
Reverted. This is another example of formatter setting worth discussing .